src/libs/polyfill/index.js

```diff
@@ -0,0 +1,18 @@
+import shouldPolyfill from './intl-polyfill';
```

Comment:
A brief description on what the file does will help covering the following checklist item.

```
The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory

```